extmod/webrepl/webrepl.py

```diff
@@ -78,3 +111,8 @@ def start_foreground(port=8266):
     stop()
     s = setup_conn(port, None)
     accept_conn(s)
+
+
+def set_ssl(flag):
+    with open('ssl_flag.py', 'wb') as sslconfig:
```

Comment:
Is there a reason to create a separate config file just for the SSL flag? Maybe I'm misunderstanding...

Reply:
This is optional I think. Although is useful to change from ws to wss without modifying `main.py`

e.g. if you include this in _main.py_ :
```
...
from ssl_flag import SSL
if SSL:
    webrepl.start(ssl=True)
else:
    webrepl.start()
```
So at the repl you can do at any time `webrepl.set_ssl(True)` or `webrepl.set_ssl(False)`
and after a soft/hard reset webrepl will start on ws or wss.